src/include/CMakeLists.txt

```diff
@@ -190,7 +191,7 @@ install (FILES kdbextension.h
 
 add_custom_target (elektra_config_headers ALL
 		   DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/kdb.h ${CMAKE_CURRENT_BINARY_DIR}/kdbconfig.h
-			   ${CMAKE_CURRENT_BINARY_DIR}/kdbversion.h)
+			   ${CMAKE_CURRENT_BINARY_DIR}/kdbversion.h ${CMAKE_CURRENT_BINARY_DIR}/kdberrors.h)
```

Comment:
Not sure, what this custom target is, but I'm pretty sure `kdberrors.h` shouldn't be in here. Even if it was included, the `${CMAKE_CURRENT_BINARY_DIR}/` part is wrong, since that is not where the header is located anymore.

Reply:
removed it